connectors/citrus-docker/src/main/java/org/citrusframework/docker/command/ContainerCreate.java

```diff
@@ -89,15 +92,15 @@ public void execute(DockerClient dockerClient, TestContext context) {
         }
 
         if (hasParameter("cmd")) {
-            if (getParameters().get("cmd") instanceof Capability[]) {
+            if (getParameters().get("cmd") instanceof String[]) {
```

Comment:
I think all those if-conditions should look something like this (with an `OR`-clause):

```suggestion
            if (getParameters().get("cmd") instanceof Capability[]
                || getParameters().get("cmd") instanceof String[]) {
```

if we just remove `Capability`, as far as I see, we lose functionality. because it is part of the underlying dockerjava api: `import com.github.dockerjava.api.model.Capability;`.

Reply:
Oh yeah I wasnt sure about those 'instanceof Capability[]' checks; they appeared to be typos for me 😅. But let me put them back

Reply:
typos don't compile 😉

Reply:
PR updated